src/components/Countdown/Countdown.tsx

```diff
@@ -0,0 +1,96 @@
+import { ComponentProps, forwardRef, useEffect, useState } from 'react';
+import { getCommonProps } from '../../utils';
+import classnames from 'classnames';
+import { SupportedLanguages } from '../../types/commonTypes';
+import { differenceInDays, differenceInHours, differenceInMinutes, differenceInSeconds } from 'date-fns';
+import { zhCN, enUS } from 'date-fns/locale';
+import { CountdownVariants } from './types';
+import { Duration } from './Duration'; // Import the Duration component
+
+// You'll need to change the ComponentProps<"htmlelementname"> to match the top-level element of your component
+export interface CountdownProps extends ComponentProps<'div'> {
+  /**
+   * The date the countdown should end
+   */
+  endDateTime: Date;
+  /**
+   * Any descriptor to appear below the coundown
+   */
+  intervalDescription?: string;
+  /**
+   * Text translation for what the countdown is for
+   */
+  label?: string;
+  /**
+   * Locale to use to format date strings
+   */
+  locale?: SupportedLanguages;
+  /**
+   * Variant of the countdown
+   */
+  variant?: CountdownVariants;
+}
+/**
+ * ## Overview
+ *
+ * Overview of this widget
+ *
+ * [Figma Link](https://www.figma.com/design/hMu9IWH5N3KamJy8tLFdyV/EASEL-Compendium%3A-Tokens%2C-Components-%26-Patterns?node-id=10850-35410&m=dev)
+ *
+ * [Storybook Link](https://phillips-seldon.netlify.app/?path=/docs/components-countdown--overview)
+ */
+const Countdown = forwardRef<HTMLDivElement, CountdownProps>(
+  (
+    {
+      endDateTime,
+      label = 'Lots Close in',
+      intervalDescription = 'Lots Close in 1-minute intervals',
```

Comment:
was it 1 or 2 minute intervals?

Reply:
supposedly 1 even though the figma says otherwise..